src/hooks/useMember/useMember.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import {
+  getMember as getMemberFn,
+  setMember as setMemberFn,
+  setActiveChannel as setActiveChannelFn,
+  removeMember as removeMemberFn,
+} from './utils'
+
+export type Member = {
```

Comment:
To avoid confusion I think we could name this differently, maybe `ActiveMember`?  Probably we will have something like `Member`  after we update queries and generate new types.